BASE/src/Microsoft.ApplicationInsights/GlobalSuppressions.cs

```diff
@@ -1,30 +1,30 @@
-﻿// <copyright file="GlobalSuppressions.cs" company="Microsoft">
-// Copyright © Microsoft. All Rights Reserved.
-// </copyright>
-//
-// To add a suppression to this file, right-click the message in the 
-// Code Analysis results, point to "Suppress Message", and click "In Suppression File".
-// You may also add suppressions to this file manually.
-
-#pragma warning disable SA1025 // Code must not contain multiple whitespace in a row
-
-using System.Diagnostics.CodeAnalysis;
-
-[assembly:SuppressMessage(
-
-            category:       "Microsoft.Usage",
-            checkId:        "CA2213: Disposable Fields Should Be Disposed",
-            MessageId =     "startRunnerEvent",
-            Scope =         "member",
-            Target =        "Microsoft.ApplicationInsights.Channel.InMemoryTransmitter.#Dispose(System.Boolean)", 
-            Justification = "startRunnerEvent is being disposed.")]
-
-[assembly:SuppressMessage(
-            category:       "Microsoft.Usage",
-            checkId:        "CA2213: Disposable Fields Should Be Disposed", 
-            MessageId =     "telemetryChannel", 
-            Scope =         "member", 
-            Target =        "Microsoft.ApplicationInsights.Extensibility.TelemetrySink.#Dispose()", 
-            Justification = "telemetryChannel is being disposed using ?. syntax")]
-
+﻿// <copyright file="GlobalSuppressions.cs" company="Microsoft">
```

Comment:
this should also simply appear as "File renamed without changes"

Reply:
i honestly have no idea about this one. the contents appear to be identical and no tests are failing (i confirmed the # of tests are the same)